src/SkynetServer/Resources/email.txt

```diff
@@ -0,0 +1,9 @@
+﻿Welcome to Skynet Messenger!
```

Comment:
Why do we have a txt **and** a XHTML file?

Reply:
[MailKit](/jstedfast/MailKit)'s `BodyBuilder` allows to create a multipart alternate MIME entity. This allows old mail clients which do not support HMTL to display a text form of the message. And even new clients treating our mail as spam might use the text message instead of HTML.